docs/reference/commandline/daemon.md

```diff
@@ -643,4 +644,20 @@ set like this:
     /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
 
 
+# Default cgroup parent
```

Comment:
@mrunalp can you provide me with an example of specifying the option in this way?